### PR TITLE
Vic notif view UI updated 

### PR DESCRIPTION
**Pull request by vmreed1**

### CODE REVIEW

File: TranquilApp/ViewControllers/NotificationViewController.swift [hunk 1]
```diff
@@ -22,29 +29,88 @@ class NotificationViewController: UIViewController {
...
+    
+    @objc private func addWaves(){
+        background.view.frame = CGRect(x: 0, y: 0, width: UIScreen.main.bounds.width, height: UIScreen.main.bounds.height)
+        background.didMove(toParent: self)
+        background.modalPresentationStyle = .fullScreen
+        view.insertSubview(background.view, at: 0)
+    }
+    
+    @objc private func addBreathingButton() {
+        breathingButton.setTitle("Press to go from notification to Breathing Exercises", for: .normal)
```
Comment (hdinh77) [inline TranquilApp/ViewControllers/NotificationViewController.swift:57]: Don't think we need the "from notification" part... "Press To Go to Breathing Exercises" seems fine

File: TranquilApp/ViewControllers/NotificationViewController.swift [hunk 2]
```diff
@@ -22,29 +29,88 @@ class NotificationViewController: UIViewController {
...
+        breathingButton.titleLabel?.numberOfLines = 0 // 0 indicates dynamic
+        breathingButton.titleLabel?.lineBreakMode = NSLineBreakMode.byWordWrapping
+        breathingButton.titleLabel?.textAlignment = .center
+        view.addSubview(breathingButton)
+        breathingButton.backgroundColor = UIColor(SECONDARY_TEXT_COLOR)
+        breathingButton.setTitleColor(.black, for: .normal)
```
Comment (hdinh77) [inline TranquilApp/ViewControllers/NotificationViewController.swift:63]: do we wanna set title color to .white to keep it consistent with the rest of the app?

File: TranquilApp/ViewControllers/NotificationViewController.swift [hunk 3]
```diff
@@ -22,29 +29,88 @@ class NotificationViewController: UIViewController {
...
-        button.frame = CGRect(x: 35, y:160, width: 330, height: 600)
-        button.layer.cornerRadius = 10
-        button.addTarget(self, action: #selector(didTapButton), for: .touchUpInside)
         
+        name.textAlignment = .center
+        name.text = "What exercise will help your anxiety right now?"
+        name.textColor = UIColor(SECONDARY_TEXT_COLOR)
+        name.frame = CGRect(x: 25, y: 100, width: 350, height: 70)
+        name.font = .systemFont(ofSize: 25, weight: UIFont.Weight(rawValue: 10))
+        name.isUserInteractionEnabled = false
```
Comment (hdinh77) [inline TranquilApp/ViewControllers/NotificationViewController.swift:38]: the background of `name` is white, can u change background to BACKGROUND_COLOR?

File: TranquilApp/ViewControllers/NotificationViewController.swift [hunk 4]
```diff
@@ -22,29 +29,88 @@ class NotificationViewController: UIViewController {
...
+    }
+    @objc private func didTapBreathingButton() {
         let vc = BreathingViewController()
         vc.modalPresentationStyle  = .popover
         navigationController?.pushViewController(vc, animated: true)
-//        navigationController?.popViewController(animated: true)
-//        present(vc, animated: true)
+    }
+    
+    @objc private func didTapJournalButton() {
```
Comment (hdinh77) [inline TranquilApp/ViewControllers/NotificationViewController.swift:100]: running into a bug where it pushes the journal view up a bit...
Screenshot 2023-03-01 at 2 36 44 PM

File: TranquilApp/ViewControllers/NotificationViewController.swift (hunk 1 above)
Comment (hdinh77) [reply TranquilApp/ViewControllers/NotificationViewController.swift:57]: ^^ with subsequent buttons as well

**Review by hdinh77 — CHANGES_REQUESTED** [review]

File: TranquilApp/ViewControllers/NotificationViewController.swift (hunk 1 above)
Comment (vmreed1) [reply TranquilApp/ViewControllers/NotificationViewController.swift:57]: yeah good idea

File: TranquilApp/ViewControllers/NotificationViewController.swift (hunk 2 above)
Comment (vmreed1) [reply TranquilApp/ViewControllers/NotificationViewController.swift:63]: also good idea

**Review by hdinh77 — APPROVED** [review]
Looks good, will fix view bug in later PR

**Pull request merged**